drivers/include/adxl345.h

```diff
@@ -92,8 +92,8 @@ enum {
  * @brief   Output Interrupt selection
  */
 enum {
-    INT1,  /**< Output interrupt on INT1 pin */
-    INT2   /**< Output interrupt on INT2 pin */
+    INT1_ACC,  /**< Output interrupt on INT1 pin */
+    INT2_ACC   /**< Output interrupt on INT2 pin */
```

Comment:
This chance is unrelated to the PR. Please provide a separate PR if you want this to be changed.

Reply:
OK, i'll create a new PR. But the CI will report errors until then. Should I just Blacklist the board for those cases?

Reply:
Really? Anyway, then we merge the new mini-PR first an rebase this one then.